app-profile-saml-jee-jsp/src/test/java/org/keycloak/quickstart/ArquillianProfileSamlJeeJspTest.java

```diff
@@ -47,8 +47,7 @@
 import java.net.URL;
 import java.util.concurrent.TimeUnit;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
Last nitpick. We should not use star imports.

Reply:
fixed